### DISCUSSION TITLE
Findings Text Sequence Number

### DISCUSSION THREAD

**Discussion by theMapManDebug** [GSA-TTS/FAC#4222] [General]
When pulling findings text data from the API, is there a sequence number for sorting the findings text in the proper order? Without it, it's very difficult to present the findings text sequentially, as presented in the actual single audit report. Thanks in advance.

**Comment by jadudm** [comment]
My sincere apologies; while we're in GH every day, we apparently miss the discussions section. We recommend using our helpdesk for questions: https://support.fac.gov/hc/en-us.

The order they are returned is the order they are recorded in the submission; we explicitly do a sort on the internal database row id when returning the results, but did not expose that number because it is not a reliable sequence. (It could change, for example, due to a database update.) The sequence is preserved, however.

We will consider adding a sequence in the future.
